src/Site.js

```diff
@@ -120,8 +119,8 @@ class Site {
 
     // Page template path
     this.pageTemplatePath = path.join(__dirname, PAGE_TEMPLATE_NAME);
-    const env = nunjucks.configure({ autoescape: false });
-    this.pageTemplate = nunjucks.compile(fs.readFileSync(this.pageTemplatePath, 'utf8'), env);
+    this.pageTemplate = njUtil.nunjucks.compile(
+      fs.readFileSync(this.pageTemplatePath, 'utf8'), njUtil.unescapedEnv);
```

Comment:
Could add a `compile` method in nunjucksUtils for this to avoid exporting `unescapedEnv` / `nunjucks`